README.md

```diff
@@ -175,9 +185,8 @@ Use https | `false` | `WLS_SECURE`
 
 ## Configure the exporter
 
-The sidecar does not have a landing page. Configuration is done by sending a PUT request to the path `/configuration`. 
-You can do this with `curl`; when used with the [operator](https://github.com/oracle/weblogic-kubernetes-operator/), 
-the operator will automatically configure the exporter. 
+You configure the sidecar by sending a PUT request to the path `/configuration`. 
+You can do this with `curl`; that is how the operator does it. 
```

Comment:
operator -> [WebLogic Server Kubernetes Operator](https://github.com/oracle/weblogic-kubernetes-operator/)

Reply:
I've already called it that once in the doc. @rosemarymarano do I need to use the full name each time?

Reply:
Nope; it's ok as "the operator."

Reply:
Not to quibble, but this is a relatively big piece of doc,  there are only two mentions of the operator, and the original usage and second usage are widely separated, it doesn't hurt to repeat in that situation (thus my comment).